docs/plugins/delete-by-query.asciidoc

```diff
@@ -127,25 +129,26 @@ The JSON response looks like this:
 [source,js]
 --------------------------------------------------
 {
-  "took" : 639,
+  "took" : 201,
   "timed_out" : false,
   "_indices" : {
     "_all" : {
-      "found" : 5901,
-      "deleted" : 5901,
+      "found" : 120,
+      "deleted" : 120,
       "missing" : 0,
       "failed" : 0
     },
     "twitter" : {
-      "found" : 5901,
-      "deleted" : 5901,
+      "found" : 120,
+      "deleted" : 120,
       "missing" : 0,
       "failed" : 0
     }
   },
   "failures" : [ ]
 }
 --------------------------------------------------
+// TESTRESPONSE[s/"took" : 639/"took" : "$body.took"/]
```

Comment:
Hmm - this ought to tail.... I'm looking for `"took": 639` to replace it but I made it 120. I'll have another look in a minute.